web/src/test/java/se/inera/intyg/webcert/web/web/controller/facade/ConfigControllerTest.java

```diff
@@ -65,5 +73,19 @@ void getDynamicLinksReturnsLinks() {
             final var response = (Map<String, DynamicLink>) configController.getDynamicLinks();
             assertTrue(response.containsKey("Test"));
         }
+
+        @Test
+        void getConfigurationReturnsBanners() {
```

Comment:
Would be good with a test that verifies that banners other than Webcert isn't included.